modules/oe_webtools_globan/oe_webtools_globan.module

```diff
@@ -7,68 +7,47 @@
 
 declare(strict_types = 1);
 
-use Drupal\Component\Utility\UrlHelper;
-use Drupal\Core\Asset\AttachedAssetsInterface;
+use Drupal\Component\Serialization\Json;
 use Drupal\Core\Cache\CacheableMetadata;
-use Drupal\Core\Url;
 
 /**
- * Implements hook_page_attachments().
+ * Implements hook_page_top().
  *
- * Attach smart loader.
+ * Attach smart loader with appropriate JSON snippet for the global banner.
```

Comment:
Let's add a comment here: We inject the globan JSON snippet in the page top area because the banner markup is injected in the same HTML element that contains the JSON. Because of this, we cannot place the JSON in the page head element.